packages/zoe/src/internal-types.js

```diff
@@ -255,13 +255,14 @@
 
 /**
  *
- * @callback CreateSeatManager 
+ * @callback CreateSeatManager
  *
  * The SeatManager holds the active zcfSeats and can reallocate and
  * make new zcfSeats.
  *
  * @param {ERef<ZoeInstanceAdmin>} zoeInstanceAdmin
  * @param {GetAssetKindByBrand} getAssetKindByBrand
+ * @param {(reason: Error) => void} shutdownWithFailure
```

Comment:
This is just my preference, but it would be nice to have the shutdownWithFailure type be a named callback itself, so we can add a comment to the type about the severity of effects and use the same type rather than remaking it each time

Reply:
```suggestion
 * @param {ShutdownWithFailure} shutdownWithFailure
```